packages/react/src/components/progress/progress.stories.tsx

```diff
@@ -0,0 +1,112 @@
+import { Progress, ProgressProps } from '@onfido/castor-react';
+import React, { useEffect, useState } from 'react';
+import { Meta, omit, reactMatrix, Story } from '../../../../../docs';
+
+const size = ['regular', 'large'] as const;
+const hideLabel = [false, true] as const;
+const children = ['', 'Custom label'] as const;
+
+export default {
+  title: 'React/Progress',
+  component: Progress,
+  argTypes: {
+    children: {
+      description: 'Optional label.',
+    },
+    value: {
+      table: {
+        defaultValue: { summary: '0' },
+      },
+    },
+    min: {
+      table: {
+        defaultValue: { summary: '0' },
+      },
+    },
+    max: {
+      table: {
+        defaultValue: { summary: '100' },
+      },
+    },
+    size: {
+      control: { type: 'radio', options: size },
+      table: {
+        type: { summary: size.toString() },
+        defaultValue: { summary: 'text' },
```

Comment:
This displays as default value "text", I believe you want to set here "regular".

Reply:
Done, thanks!